src/main/ui/conversation-control.gd

```diff
@@ -0,0 +1,10 @@
+extends Control
+"""
+Displays UI elements for having a conversation.
+"""
+
+"""
+Starts a new conversation.
```

Comment:
Use the other comment, as it's more descriptive: `Animates the conversation label to gradually reveal the specified text.`

Reply:
Improved comment.